utils/dbus/files/dbus.init

```diff
@@ -3,15 +3,32 @@
 
 START=60
 
-SERVICE_PID_FILE=/var/run/dbus.pid
+USE_PROCD=1
+PROG=/usr/bin/dbus-daemon
+PIDFILE=/var/run/dbus.pid
 
-start() {
+# To enable this makes only sense if the dbus has been compiled with
+# the option CONFIG_DBUS_VERBOSE=y
+
+#DEBUG=1
+
+start_service() {
 	mkdir -m 0755 -p /var/lib/dbus
 	mkdir -m 0755 -p /var/run/dbus
+
 	[ -x /usr/bin/dbus-uuidgen ] && /usr/bin/dbus-uuidgen --ensure
-	service_start /usr/bin/dbus-daemon --system
+
+	procd_open_instance
+	procd_set_param command "${PROG}"
+	procd_append_param command --system
+	procd_append_param command --nofork
+	[ -n "$DEBUG" ] && procd_set_param env DBUS_VERBOSE=1
+	procd_set_param stdout 1
+	procd_set_param stderr 1
+	procd_set_param pidfile "$PIDFILE"
```

Comment:
so my understanding of this is that it's for daemons that use a pidfile but don't create it themselves. Since dbus does this, this is unneeded.